xonsh/ptk2/key_bindings.py

```diff
@@ -339,6 +340,11 @@ def wrap_cursor_forward(event):
         b.cursor_left(count=abs(relative_begin_index))
         b.cursor_down(count=1)
 
+    @handle(Keys.ControlM, filter=IsSearching())
+    @handle(Keys.ControlJ, filter=IsSearching())
```

Comment:
Not sure how this translates to ControlR, but I won't question it :)

Reply:
ahh, but ControlR was working fine, it was that pesky Enter/Return key that was misbehaving

Reply:
Oh.....I see. Uggh